.github/bin/get-s3-image.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+#####
+# This script looks for an image in OpenStack and if not found, downloads from
+# S3 bucket, and then uploads to OpenStack
+#####
+
+set -ex
+
+image_name=$1
+bucket_name=$2
+echo "Checking if image $image_name exists in OpenStack"
+image_exists=$(openstack image list --name "$image_name" -f value -c Name)
+
+if [ "$image_exists" == "$image_name" ]; then
+    echo "Image $image_name already exists in OpenStack."
+else
```

Comment:
What if two images with the same name exist? The `openstack image list` command will return a list of two images which wont be equal to `$image_name`. Might be safer to just check that `[[ ! -n $image_exists ]]`. Quotes around variable can also be removed.